tempesta_fw/http.c

```diff
@@ -118,8 +118,8 @@ tfw_http_establish_skb_hdrs(TfwHttpMsg *hm)
 		TfwHttpHdr *hdr = hm->h_tbl->tbl + i;
 		if (hdr->skb)
 			continue;
-		if (!hdr->field.ptr)
-			break;
+		if (!hdr->field.chunks)
+			break;  /* XXX: why do we stop here? */
```

Comment:
Long time ago headers were placed one after other, so we had continous headers array and empty header meant end on the array. Not is not the case and the break must be fixed.